woocommerce/woocommerce
# Add Sortable component accessibility

### All Submissions:






### Changes proposed in this Pull Request:

* Adds the ability to reorder sortable lists using only a keyboard
* Adds focal events for tabbing through selected items
* Announces actions when using keyboard interactions

### How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Navigate to the `Sortable` component
3. Use the tab key to focus the list
4. Use up/down arrow keys to navigate the list
5. Use the space key to drop an item in the list
6. Use the escape key to cancel reordering an item
7. Enable your screen reader and open Safari
8. Test the reordering again, noting actions are announced appropriately
9. Check that the alt attributes are read for items under the `ImageGallery` component

### Other information:




### FOR PR REVIEWER ONLY:
